ceph.spec.in

```diff
@@ -183,6 +183,7 @@ BuildRequires:	keyutils-libs-devel
 BuildRequires:	libibverbs-devel
 BuildRequires:  openldap-devel
 BuildRequires:  openssl-devel
+BuildRequires:  CUnit-devel
```

Comment:
@smithfarm does this work for SLE? or do you want to WITH_SPDK=ON for the build also?